README.md

```diff
@@ -102,3 +102,7 @@ The UI can also be disabled via a runtime flag:
 ```bash
 temporalite start --headless
 ```
+
+### Known Issues
```

Comment:
Nit: this should probably be a top-level heading rather than nested under configuration.

```suggestion
## Known Issues
```